test/files/stream.iced

```diff
@@ -1,13 +1,16 @@
+{make_esc} = require('iced-error')
 crypto = require('crypto')
-stream = require('../../src/stream.iced')
-to_buffer = require ('../../src/stream_to_buffer.iced')
-enc = require('../../src/encoding.iced')
+util = require('keybase-chunk-stream').util
+basex = require('../..')
+stream = basex.stream
+enc = basex.encoding
 
 #==========================================================
 #Helper functions/constants
 #==========================================================
 
-loop_limit = 1024**2
+giant_file = 524288
+loop_limit = 8192
```

Comment:
`test_{consistency,output}` take about 3 seconds each, `test_giant_file_{consistency,output}` take about 10 seconds each. There's a ticket in for improving this performance but it will probably be fairly involved